Dockerfile.fpm

```diff
@@ -0,0 +1,30 @@
+# Build the binaries in larger image
```

Comment:
this is a dup of Dockerfile - have a look at
https://github.com/istio/fortio/blob/master/release/Dockerfile.in
and "make release" to extend/use the same pattern ?